app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
Ok, since this would require breaking existing route ('/hosts/:host_id') I will do that in a separate pull request. This kind of changes should really be done in a clean refactoring separate commit. Changes needs to be done in CLI as well.